woocommerce/woocommerce-ios
# Update WordPress-Editor-iOS to fix crash in Aztec

Closes: #11770 


## Description

There's a crash reported in peaMlT-p5-p2 due to the attempt to render Gutenberg comments in product descriptions to images. This PR adds the [update](https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1383) in the WordPress-Editor-iOS library to fix this crash.

## Testing instructions

- On wp-admin, open an existing product or create a new one.
- In the product description field, switch the classic editor to `Text` mode and add a Gutenberg comment like so:

<img width="675" alt="Screenshot 2024-01-31 at 17 18 57" src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/5449d053-7149-4093-ba14-3cbb088308a1">

- Save the change and refresh the product list on the app if needed.
- Open the updated product and tap the description field.
- Confirm that the app doesn't crash.

## Screenshots

N/A

---
